tools/cool_to_hic/cool_to_hic.xml

```diff
@@ -0,0 +1,75 @@
+<tool name="cool_to_hic" id="cool_to_hic" version="0.0.8" profile="22.05">
```

Comment:
Maybe the id as hictk? So we can extend it later for more conversions?

Reply:
done.
name and id are identical for toolfactory tools